crates/api_common/src/site.rs

```diff
@@ -72,6 +72,7 @@ pub struct CreateSite {
   pub open_registration: bool,
   pub enable_nsfw: bool,
   pub auth: String,
+  pub community_creation_admin_only: bool,
```

Comment:
Side point, but since all of these have defaults, I can probably make them optional.